srcpkgs/xbps-triggers/files/dkms

```diff
@@ -21,19 +21,23 @@ UPDATE="$5"
 export PATH="usr/bin:usr/sbin:/usr/sbin:/usr/bin:/sbin:/bin"
 
 remove_modules() {
-	local _modver _kver
+	local modname modver kver arch status
 
 	# Remove the specified modules from all kernels.
 	set -- ${dkms_modules}
 	while [ $# -gt 0 ]; do
-		$DKMS status -m "$1" | while read line; do
-			if $(echo "$line" | egrep -vq '(added|built|installed)'); then
-				shift 2; continue
+		$DKMS status -m "$1" | while IFS=':/, ' read modname modver kver arch status; do
```

Comment:
`IFS=':/, '` makes the cut a bit more aggressive than my earlier proposal, but this is probably fine. The two-stage cut in my proposal restricted the more generic `IFS='/, '` cut to only two fields (module name and version), then applied the more specific `IFS=', '` cut to only the remainder. Is there any legitimate concern that any of the fields might have characters ':' or '/' that would foil this split?

Reply:
Good point. I'd never thought if the `:` and `,` characters might appear in a module's name.

None of the fields can have the `/` character, as that would break dkms itself. I've verified that.

I'm more concerned about the `:` and `,` characters. They have the possibility to break this script.

I've pushed a fix in the latest commit.

Reply:
**UPDATE:**
No, there is no concern that `,` `:` `/` characters might appear in any of the fields.  
`dkms` itself assumes so.

https://github.com/dell/dkms/blob/ea97ad27efb4d227242e322f9d30c7280a6eba47/kernel_prerm.d_dkms#L7-L11